bf2fc6cc711aee1a0c2a/kas-fleet-manager
# remove handling of api-guidelines rules from private agent endpoints

## Description

- The first commit remove handling of agent_clusters and agent-clusters endpoint. The resulting change only handles agent-clusters endpoint
- Second commit: removes handling of deprecated fields and revert back to using camelCase

## Verification Steps


1. Integration test should continue to pass
2. Run the API before taking into account this change and deploy a kafka
3. Wait for the kafka to be ready and run the API again with the changes in the PR
4. Make sure that the kafka stays ready
5. Attempt to delete the previously deployed kafka and make sure it is deleted
6. create a new kafka and make sure it goes into a ready state

## Checklist (Definition of Done)
